docs/modules/user-guide/partials/proc_adding-components-to-a-devfile.adoc

```diff
@@ -111,6 +126,12 @@ Each component in a single devfile must have a unique name and use one of the ob
 |string
 |no
 |The path in the component container where you mount the volume.
+|===
+
+.volume object
+[cols="1,1,1,1"]
+|===
+|Key |Type| Required| Description
```

Comment:
I find it difficult the words **Key** and **Type** in the table header because these have multiple possible meanings. What do they mean here and how are they used in the devfile?